src/gateways/paypal.js

```diff
@@ -12,19 +12,18 @@ Paypal.prototype.addChildPublicKey = function() { };
 
 Paypal.prototype.createToken = function(form, clickPromise) {
     var request = new XMLHttpRequest();
-    request.open('GET', '/payment/token/paypal');
+    request.open('GET', 'https://www.mocky.io/v2/5d9c819f31000037c92fc621');
     request.onload = function() {
-        var response = JSON.parse(request.response);
         if (request.status == 200) {
+            var response = JSON.parse(request.response);
             clickPromise.finish(request.status, {content: response, type: this.type, name: this.name});
             return;
         }
 
-        clickPromise.finish(request.status, {name: this.name, cause: response.ErrorReport});
+        clickPromise.finish(request.status, {name: this.name, cause: 'error', type: this.type});
     }.bind(this);
 
     request.onerror = function() {
-        console.log(request);
         clickPromise.finish(request.status, {name: this.name, cause: 'error'});
```

Comment:
Vamos aproveitar e corrigir isso igual fizemos pra mundipagg?
```suggestion
        clickPromise.finish(request.status, {name: 'paypal', cause: 'error'});
```
![Screen Shot 2019-10-09 at 21 00 41](https://user-images.githubusercontent.com/22858307/66529085-66cc2080-ead8-11e9-86d7-a6ca48568df5.png)